src/components/pagination/pagination_button.styles.ts

```diff
@@ -22,19 +17,9 @@ export const euiPaginationButtonStyles = (euiThemeContext: UseEuiTheme) => {
   const halfSizeM = mathWithUnits(euiTheme.size.m, (x) => x / 2);
   const disabled = euiButtonEmptyColor(euiThemeContext, 'disabled');
 
-  // && to increase specificity. Can likely be removed once EuiButtonEmpty has been converted.
-
   return {
     // Base
-    euiPaginationButton: css`
-      && {
-        ${fontSizeS}
-        padding: 0;
-        ${logicalTextAlignCSS('center')}
-        border-radius: ${euiTheme.border.radius.medium};
-        outline-offset: -${euiTheme.focus.width};
-      }
-    `,
+    euiPaginationButton: css``,
```

Comment:
It looks like we still need the outline-offset - the focus/tab outline is cut off in staging where it's not in prod/prior to the Emotion conversion
```suggestion
    euiPaginationButton: css`
      outline-offset: -${euiTheme.focus.width};
    `,
```